imports/ui/components/shows/show_page.js

```diff
@@ -24,20 +24,18 @@ Template.showPage.onCreated(function() {
                 true, function(error, result) {
                 if (!error && result)
                   Session.set('currentPlaylist', result);
-                });
-              }
+              });
             }
+          }
         });
       }
     });
   });
 });
 
 Template.showPage.helpers({
-  show: ()=>  Shows.findOne({ slug: FlowRouter.getParam('slug') }),
+  show: () =>  Shows.findOne({}),
```

Comment:
Again, we could have multiple shows in scope here, which would lead to unreliable retrieval of the show data.